quickstarts/sentry-software/config.yml

```diff
@@ -0,0 +1,44 @@
+slug: sentry-software
```

Comment:
I would rather use the product name (hardware-sentry) instead of the company name who develops the product (sentry-software)